daemon/log.rs

```diff
@@ -63,3 +65,27 @@ pub fn clean_log_handles(task_id: usize, settings: &Settings) {
         );
     };
 }
+
+/// Return stdout and stderr of a finished process
+/// Everything is compressed using Brotli and then encoded to Base64
+pub fn read_and_compress_log_files(
+    task_id: usize,
+    settings: &Settings,
+) -> Result<(Vec<u8>, Vec<u8>)> {
+    let (mut stdout_handle, mut stderr_handle) = match get_log_file_handles(task_id, settings) {
+        Ok((stdout, stderr)) => (stdout, stderr),
+        Err(err) => return Err(anyhow!(format!("Error while opening the output files: {}", err))),
```

Comment:
I think you can use [`bail!()`](https://docs.rs/anyhow/1.0.28/anyhow/macro.bail.html) here

Reply:
Nice. Didn't know about that one!